src/service/iot_config.proto

```diff
@@ -365,6 +365,53 @@ message admin_remove_key_req_v1 {
 
 message admin_key_res_v1 {}
 
+message gateway_info {
+  /// The asserted h3 (string) location of the gateway.
+  // Empty string if the gateway is not found
+  string location = 1;
+  bytes address = 2;
+  bytes owner = 3;
+  bool is_full_hotspot = 4;
+  /// the transmit gain value of the gateway in dbi x 10
+  // For example 1 dbi = 10, 15 dbi = 150
+  int32 gain = 5;
+  region region = 6;
+  blockchain_region_params_v1 region_params = 7;
+}
+
+enum gateway_error_reason {
+  reason_none = 0;
+  gateway_not_found = 1;
+}
+
+message gateway_error_res {
+  bytes address = 1;
+  gateway_error_reason error = 2;
+}
+
+/// Look up the details of a given hotspot public key
+message gateway_info_req_v1 {
+  /// The pubkey_bin address of the gateway to look up
+  bytes address = 1;
```

Comment:
this may need a request signature depending on how we want to setup the network around the config service and between it and the other oracles; we may also need a timestamp on the request for the purposes of rate-limiting potential non-oracle users. thoughts @madninja ?

Reply:
1. Timestamp isn't going to help with rate limiting
2. Signing the request and response sounds like a good idea.. We should talk through whether 
    - This info service is open to the public, in which case only the response needs to be signed by the config service so the caller can confirm it came from the config service
    - This info service is private to a set of approved keys, in which case the request is signed by the caller and verified against approved keys. The  response is signed by the config service

I believe we should limit access since listing hotspots is a wildly popular script

Reply:
addressed here: e8f3a601a1ff0713430ee9e1b70406f3efeb39ff